core/dbt/task/debug.py

```diff
@@ -126,6 +144,35 @@ def run(self):
     def interpret_results(self, results):
         return results
 
+    def _get_adapter(self, profile):
+        """Using profile, return the adapter instance."""
+        register_adapter(profile)
+        return get_adapter(profile)
+
+    def _read_adapter_info(self, cls):
+        """Read the adapter name and version from the setup.py file of the adapter plugin."""
+        current_adapter_plugin = next(
+            filter(lambda x: x.adapter == cls, get_adapter_plugins(None))
+        )
+        adapter_setup_script = current_adapter_plugin.include_path / Path("../../../setup.py")
```

Comment:
Well, I don't love this for a number of reasons but this _is_ the only idea out of 5 I had to get this working. If you have a suggestion, let me know. I dug around the objects/classes available to the `DebugTask` and found nothing.

Reply:
I looked hard into this but the method used here is not truly installation agnostic. It's really difficult to test locally because it won't work with `-e` pip installs. It won't find adapters for beta versions of dbt either. I think what I have here is better for that reason...even if I don't love it. I'm going to split the difference and steal it's way of obtaining the `version`.

Reply:
I _did_ find something far better 
```
module_name = f"{'.'.join(cls.__module__.split('.')[:-1])}.__version__"
version = importlib.import_module(module_name).version
```